lib/utils/node_utils.dart

```diff
@@ -127,7 +128,12 @@ class NodeUtils {
 
     if (unreceivedBlocks.isNotEmpty) {
       for (AccountBlock unreceivedBlock in unreceivedBlocks) {
-        sl<AutoReceiveTxWorker>().addHash(unreceivedBlock.hash);
+        if (sharedPrefsService!.get(
```

Comment:
Consistency: Is there are reason this check is done here but not on line 177?

Reply:
No, I don't think so. Although the `autoReceive` does do the check, it's better to prevent to call all together.